conda/install.py

```diff
@@ -397,23 +405,49 @@ def replace_long_shebang(mode, data):
     return data
 
 
+def replace_entry_point_shebang(all_data, placeholder, new_prefix):
+    """Code adapted from pyzzer.  This is meant to deal with entry point exe's created by distlib, which
```

Comment:
Good call @takluyver - this was "adapted" in the sense that Melania Trump "adapted" Michelle Obama's speech.  Adding the copyright notice was appropriate.